lib/Doctrine/Common/Util/Debug.php

```diff
@@ -109,10 +109,10 @@ public static function export($var, $maxDepth)
                 }
             } else if ($isObj) {
                 $return = new \stdclass();
-                if ($var instanceof \DateTime) {
-                    $return->__CLASS__ = "DateTime";
+                if ($var instanceof \DateTime || $var instanceof \DateTimeInterface) {
```

Comment:
Can actually simply check against `DateTimeInterface`, since we don't support older PHP versions anymore